Makefile

```diff
@@ -14,5 +14,5 @@ clean:
 
 test:
 	rm -rf _build/_tests
-	dune build test test-bin @install
+	dune build test test-bin examples @install
```

Comment:
The example builds are all commented out by default, but it's useful to uncomment them for testing sometimes.